yarl/_url.py

```diff
@@ -1023,6 +1024,16 @@ def update_query(self, *args, **kwargs):
             self._val._replace(query=self._get_str_query(query) or ""), encoded=True
         )
 
+    def without_query_params(self, *keys: str) -> "URL":
```

Comment:
Let's use something like `query_params` since `keys` is ambiguous. There's no concept called “keys” in the HTTP specs.